module/VuFind/src/VuFind/Search/NewItemsHelper.php

```diff
@@ -144,16 +138,16 @@ public function getFundList()
      *
      * @return array
      */
-    public function getHiddenFilters()
+    public function getHiddenFilters(): array
     {
-        if (!isset($this->config->filter)) {
+        if (! isset($this->config['filter'])) {
```

Comment:
Minor style suggestion: we don't put spaces after `!`:

```suggestion
        if (!isset($this->config['filter'])) {
```

Reply:
Though alternatively, I have a feeling that this entire function could be reduced to:

```
return (array)($this->config['filter'] ?? []);
```

Now that we're not using the objects, a bit of typecasting and null coalescing should cover all the same edge cases -- though feel free to leave it as-is if you don't confidently agree with me and/or if there's not already adequate test coverage of the possible edge cases (string value, array value, no value defined).